### DISCUSSION TITLE
Modifying Specific Projects Example...

### DISCUSSION THREAD

**Discussion by DynConcepts** [3F/MvsSln#41] [Q/A]
https://github.com/3F/MvsSln/issues/3

Shows how to modify a Solution, I am looking for how to modify a Project... e.g. Change the $OutDir property and update the (.vcxproj in this case) file...

**Comment by 3F** [comment]
IXProject in **2.x** wraps an msbuild, so you can simply use it separately or such as

```csharp
IXProject xp = ...
xp.SetProperty("OutDir", "new-value");
xp.Save();
```

https://github.com/3F/MvsSln/blob/9ce5885282d65ee32bfc03808c83c1113c1928e3/MvsSln/Core/IXProject.cs#L186-L191

easy to group like

```csharp
xp.SetProperties(new Dictionary<string, string>()
{ 
    { "name1", "value1" },
    { "name2", "value2" },
    { "name3", "value3" }
});
```

```xml
  <PropertyGroup>
    <name1>value1</name1>
    <name2>value2</name2>
    <name3>value3</name3>
  </PropertyGroup>
```

etc.

**Comment by DynConcepts** [comment]
Close but a related problem has come up... This is covered in Issue #39...

**Comment by 3F** [comment]
I think it's time to play with new [Discussions](https://github.com/3F/MvsSln/discussions) for the similar general questions like this.

@DynConcepts Please use *Discussions* for cases like this instead of opening new *Issues*. We'll test it together. Thanks.
